pkg/synthetictests/disruption.go

```diff
@@ -117,6 +117,39 @@ func testServerAvailability(
 	}
 }
 
+func TestAPIServerIPTablesAccessDisruption(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
+	const testName = "[bz-kube-apiserver] kube-apiserver should be accessible by clients using internal load balancer"
+	namespacesToCount := map[string]int{}
+	messages := []string{}
+	for _, event := range events {
+		reason := monitorapi.ReasonFrom(event.Message)
+		if !strings.Contains(reason, "iptables-operation-not-permitted") {
+			continue
+		}
+		ns := monitorapi.NamespaceFromLocator(event.Locator)
+		namespacesToCount[ns] = namespacesToCount[ns] + 1
+		messages = append(messages, event.String())
+	}
+
+	var tests []*junitapi.JUnitTestCase
+	test := &junitapi.JUnitTestCase{Name: testName}
+	if len(messages) > 0 {
+		failureOutput := ""
+		for _, ns := range sets.StringKeySet(namespacesToCount).List() {
+			failureOutput += fmt.Sprintf("namespace/%v has %d instances of 'write: operation not permitted'\n", ns, namespacesToCount[ns])
+		}
+		failureOutput += "\n\n"
+		failureOutput += strings.Join(messages, "\n")
+
+		test.FailureOutput = &junitapi.FailureOutput{
+			Output: failureOutput,
```

Comment:
this looks like a non-flake, fatal error?  Current estimates of failure rates are >1%, so perhaps it's worth opening as a flake, and then tightening to fatal if/when existing releases covered by the origin dev branch (which currently includes older 4.13.z, etc, for runs testing 4.13.z -> 4.14 nightly updates) are sufficiently clean?

Reply:
I did forget unconditionally add a success.  I'll do so